Packs/SymantecEndpointSecurity/Integrations/SymantecEndpointSecurity/SymantecEndpointSecurity.py

```diff
@@ -0,0 +1,471 @@
+import itertools
+import demistomock as demisto
+from urllib3 import disable_warnings
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from datetime import datetime
+import dateparser
+import time
+
+disable_warnings()
+
+
+# CONSTANTS
+VENDOR = "symantec"
+PRODUCT = "endpoint_security"
+DEFAULT_CONNECTION_TIMEOUT = 30
+MAX_CHUNK_SIZE_TO_READ = 1024 * 1024 * 150  # 150 MB
+
+
+class UnauthorizedToken(Exception):
+    """
+    Exception raised when the authentication token is unauthorized.
+    """
+
+    ...
+
+
+class NextPointingNotAvailable(Exception):
+    """
+    Exception raised when the next pointing is not available.
+    """
+
+    ...
+
+
+class Client(BaseClient):
+    def __init__(
+        self,
+        base_url: str,
+        token: str,
+        stream_id: str,
+        channel_id: str,
+        verify: bool,
+        proxy: bool,
+        fetch_interval: int,
+    ) -> None:
+
+        self.headers: dict[str, str] = {}
+        self.token = token
+        self.stream_id = stream_id
+        self.channel_id = channel_id
+        self.fetch_interval = fetch_interval
+
+        super().__init__(
+            base_url=base_url,
+            verify=verify,
+            proxy=proxy,
+            timeout=180,
+        )
+
+        self._update_access_token()
+
+    def _update_access_token(self):
+        """
+        Retrieves an access token using the `token` provided in the params.
+        """
+        get_token_headers: dict[str, str] = {
+            "accept": "application/json",
+            "content-type": "application/x-www-form-urlencoded",
+            "Authorization": f"Basic {self.token}",
+        }
+        try:
+            res = self._http_request(
+                "POST",
+                url_suffix="/v1/oauth2/tokens",
+                headers=get_token_headers,
+                data={},
+            )
+        except Exception as e:
+            raise DemistoException("Failed getting an access token") from e
+
+        if "access_token" not in res:
+            raise DemistoException(
+                f"The key 'access_token' does not exist in response, Response from API: {res}",
+                res=res,
+            )
+        self.headers = {
+            "Authorization": f'Bearer {res["access_token"]}',
+            "Accept": "application/x-ndjson",
+            "Content-Type": "application/json",
+            "Accept-Encoding": "gzip",
+        }
+
+    def get_events(self, payload: dict[str, str]) -> str:
```

Comment:
let's add the parsing code to the end of this method (and remove `parse_raw_event_response`), since we'll never want the response raw, and it's in an unusual format that doesn't make sense to parse separately.